app/controllers/api/v1/plans_controller.rb

```diff
@@ -87,6 +87,22 @@ def create
       # rubocop:enable Metrics/AbcSize, Metrics/CyclomaticComplexity
       # rubocop:enable Metrics/PerceivedComplexity, Metrics/MethodLength
 
+      # GET /api/v1/plans
+      def index
+        # ALL can view: public
```

Comment:
Thanks for putting this outline of permissions here. I think it will be really useful going forward when we're trying to debug issues